cmd/cconfig/main.go

```diff
@@ -38,14 +39,15 @@ type Command struct {
 	Ctx   interface{}
 }
 
-var usage = `usage: reborn-config  [-c <config_file>] [-L <log_file>] [--log-level=<loglevel>]
-		[--http-addr=<debug_http_addr>] [--pidfile=<file>] <command> [<args>...]
+var usage = `usage: reborn-config [options]
+
 options:
-   -c	set config file
-   -L	set output log file, default is stdout
-   --log-level=<loglevel>	set log level: info, warn, error, debug [default: info]
-   --http-addr=<debug_http_addr>  debug http address
-   --pidfile=<file>  program pidfile
+   -c								set config file
+   -L								set output log file, default is stdout
+   --log-level=<loglevel>			set log level: info, warn, error, debug [default: info]
+   --http-addr=<debug_http_addr>	debug http address
+   --pidfile=<file>					program pidfile
+   --server-auth=PASSWORD			backend server password
```

Comment:
putting the server password in config.ini as reborn env may be better?

server password is not only for config, but also for proxy, so I think it can be set in env.